test/overlays/knative/overlays/kourier/net_kourier_controller_deployment_patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: net-kourier-controller
+  namespace: knative-serving
+spec:
+  template:
+    spec:
+      containers:
+        - name: controller
+          resources:
+            limits:
+              cpu: 100m
+              memory: 128Mi
+            requests:
+              cpu: 5m
+              memory: 32Mi
```

Comment:
I think we should consider migrating to use knative operator to configure these settings without using kustomize overlays

Reply:
Can we do it in a separate PR ?

Reply:
Created the issue https://github.com/kserve/kserve/issues/2982